client/verta/verta/_uac/_workspace.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+from verta._protos.public.uac import WorkspaceV2_pb2 as _Workspace
+
+class Workspace(object):
+    """
+    Object representing a Workspace.
+
+    """
+
+    def __init__(self, conn, msg):
+        self.conn = conn
+        self.msg = msg
+        self.id = msg.id
+        self.org_id = msg.org_id
+        self.name = "{}:{}".format(msg.org_id, msg.name)
```

Comment:
I saw [in Slack](https://vertaai.slack.com/archives/C03PWNZU5CM/p1673376588729409?thread_ts=1673369535.993669&cid=C03PWNZU5CM) that we use `%org_id/%workspace_name`. Should this be

```suggestion
        self.name = "{}/{}".format(msg.org_id, msg.name)
```

instead?

(I don't think it matters too much. We can change later)

Reply:
no, the problem is that "/" doesn't work for grpc proxy. Need to use some other symbol (":" is allowed too currently) or implement your suggestion https://vertaai.slack.com/archives/C03PWNZU5CM/p1673390118488849?thread_ts=1673381601.789169&cid=C03PWNZU5CM